application-admintools-default/src/main/resources/templates/wikiSizeTemplate.vm

```diff
@@ -41,11 +43,24 @@
                 <li>$escapetool.xml($services.localization.render('adminTools.dashboard.instanceUsage.wiki.attachments',
                   [$wikiData.getAttachmentsCount(), $wikiData.getAttachmentsSize()]))</li>
               </ul>
-          #end
+            #end
+          </ul>
+        #else
+          #error($escapetool.xml($services.localization.render('adminTools.dashboard.instanceUsage.error')))
+        #end
+      </div>
+      <hr/>
+      <div class="specific-stats">
+        #set ($solutionLink = $xwiki.getURL('AdminTools.SpamPages'))
```

Comment:
This is not used

Reply:
Sorry, I got it confused with another PR and I thought I removed it from here too.